compiler/src/iree/compiler/Codegen/Dialect/Codegen/IR/IREECodegenInterfaces.td

```diff
@@ -129,8 +129,8 @@ def IREECodegen_LoweringConfigAttrInterface :
   ];
 }
 
-def IREECodegen_PackedLayoutAttrInterface :
-  AttrInterface<"PackedLayoutAttrInterface"> {
+def IREECodegen_PackedLayoutAttr :
```

Comment:
Maybe we should use `PackedLayoutMaterializerAttr` instead, similar to the `LayoutAttrInterface -> LayoutMaterializerAttr` renaming as this interface is materialization focused as well?